druid/examples/multiwin.rs

```diff
@@ -78,7 +80,58 @@ fn ui_builder() -> impl Widget<State> {
     row.add_child(Padding::new(5.0, inc_button));
     row.add_child(Padding::new(5.0, dec_button));
     col.add_flex_child(Align::centered(row), 1.0);
-    col
+    Glow::new(col)
+}
+
+struct Glow<W> {
+    inner: WidgetPod<State, W>,
+}
+
+impl<W: Widget<State>> Glow<W> {
+    pub fn new(inner: W) -> Glow<W> {
+        Glow {
+            inner: WidgetPod::new(inner),
+        }
+    }
+}
+
+impl<W: Widget<State>> Widget<State> for Glow<W> {
+    fn event(&mut self, ctx: &mut EventCtx, event: &Event, data: &mut State, env: &Env) {
+        self.inner.event(ctx, event, data, env);
+    }
+
+    fn lifecycle(&mut self, ctx: &mut LifeCycleCtx, event: &LifeCycle, data: &State, env: &Env) {
+        if matches!(event, LifeCycle::HotChanged(_)) {
+            ctx.request_paint();
+        }
+        self.inner.lifecycle(ctx, event, data, env);
+    }
+
+    fn update(&mut self, ctx: &mut UpdateCtx, old_data: &State, data: &State, env: &Env) {
+        if old_data.glow_hot != data.glow_hot {
+            ctx.request_paint();
+        }
+        self.inner.update(ctx, data, env);
+    }
+
+    fn layout(
+        &mut self,
+        ctx: &mut LayoutCtx,
+        bc: &BoxConstraints,
+        data: &State,
+        env: &Env,
+    ) -> Size {
+        let size = self.inner.layout(ctx, bc, data, env);
+        self.inner.set_layout_rect(size.to_rect());
+        bc.max()
```

Comment:
Returning size would make this container 'neutral' regarding layout, which seems more idiomatic to me
```suggestion
        size
```

Reply:
Probably better with `size` indeed.